SFML/SFML
# Fixed the installation of pkg-config files

## Description

Fixes #1433 and #1075 by removing the duplicated pkg-config install step and making adjustments to the Config.cmake.

- Set `SFML_INSTALL_PKGCONFIG_FILES` to `True` if you want the pkg-config files installed
- Set `SFML_PKGCONFIG_INSTALL_PREFIX` to a custom directory or use the default value:
    - On FreeBSD or OpenBSD it's set to `${CMAKE_INSTALL_PREFIX}/libdata/pkgconfig`
    - Otherwise the default is `${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig`

## How to test this PR?

Install SFML with CMake on any system and check that it works as expected.

@AMDmi3 and @jcowgill could you review this? Thanks! 🙂
